libmscore/sym.cpp

```diff
@@ -5714,6 +5719,12 @@ const std::array<const char*, int(SymId::lastSym)+1> Sym::symUserNames = { {
 
 //    additional symbols
 
+      "Parenthesised double flat accidental",
+      "Parenthesised flat accidental",
+      "Parenthesised natural accidental",
+      "Parenthesised sharp accidental",
+      "Parenthesised double sharp accidental",
```

Comment:
I think we'd need them embedded in `QT_TRANSLATE_NOOP("symUserNames", "...")`

Reply:
Do we need them translated though?

Reply:
(And the 4 below with a better descriptive string maybe)

Reply:
more generally, is there a rule for which glyph we translate and which ones we don't?

Reply:
All that are visible in the UI, like the palettes (except for the symbols palette) or in the status bar

Reply:
They shouldn't be exposed to the ui.

except maybed for the master palettes?